modules/local/variant_extractor/main.nf

```diff
@@ -4,7 +4,7 @@ process VARIANT_EXTRACTOR {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'oras://community.wave.seqera.io/library/htslib_pysam_tabix_pip_variant-extractor:a12ef217eccf6ba8':
+        'https://community.wave.seqera.io/library/htslib_pysam_tabix_pip_variant-extractor:a12ef217eccf6ba8':
```

Comment:
I don't think this is the correct adress. 

```
singularity pull https://community.wave.seqera.io/library/htslib_pysam_tabix_pip_variant-extractor:a12ef217eccf6ba8
WARNING: Authentication token file not found : Only pulls of public images will succeed
FATAL:   The requested image was not found in the library
```

Reply:
Then, I think it is not possible to convert oras to https for wave containers. I have been checking https://seqera.io/containers/
<img width="780" alt="Screenshot 2024-11-28 at 15 27 22" src="https://github.com/user-attachments/assets/141490df-5181-40bd-b5f0-bc29296631d1">

and found out this.

Do you have another idea to create mulled images other than seqera containers?

Reply:
thanks for the link! new singularity images  are working for me now. Can you please review it again? Thanks